howso.amlg

```diff
@@ -1430,6 +1483,64 @@
 				(conclude (call !return (assoc errors (list time_series_prep_map))) )
 			)
 
+			(if (size continue_series_values)
+				(let
+					(assoc original_continue_series_features continue_series_features)
+
+					(if (= 0 (size continue_series_features))
+						(conclude (conclude
+							(call !return (assoc
+								errors (list "continue_series_values is provided without continue_series_features, please specify continue_series_features")
+							))
+						))
+					)
+
+					(assign (assoc
+						continue_series_values
+							(map
+								(lambda (let
+									(assoc
+										continue_values (current_value 1)
+										trainee_clone (concat trainee (rand))
+									)
+									(create_entities trainee_clone (null))
```

Comment:
maybe not wildly important, but should there be some check to be sure this entity does not already exist? (if (rand) gave the same number twice? ...... horribly unlikely, but maybe worth preparing for)